build-tools/installers/create-installers.targets

```diff
@@ -159,6 +159,7 @@
     <_MSBuildFiles Include="$(MSBuildSrcDir)\LayoutBinding.cs" />
     <_MSBuildFiles Include="@(AndroidSupportedTargetJitAbi->'$(MSBuildSrcDir)\lib\%(Identity)\libmono-android.debug.so')"       Condition=" '$(PackageId)' != 'Microsoft.Android.Sdk' " />
     <_MSBuildFiles Include="@(AndroidSupportedTargetJitAbi->'$(MSBuildSrcDir)\lib\%(Identity)\libmono-android.release.so')"     Condition=" '$(PackageId)' != 'Microsoft.Android.Sdk' " />
+    <_MSBuildFiles Include="@(AndroidSupportedTargetJitAbi->'$(MSBuildSrcDir)\lib\%(Identity)\libmonodroid-mono-api.so')"       Condition=" '$(PackageId)' != 'Microsoft.Android.Sdk' " />
```

Comment:
Naming "quibble": we've been "trying" (kinda/sorta) to keep "monodroid" out of filenames, and nominally been successful:

```bash
% find /Library/Frameworks/Xamarin.Android.framework/Versions/Current -iname \*monodroid\*
# No matches
```

(Within `.apk` naming is another matter…)

`src/monodroid` creates `libmono-android*` files, not `libmonodroid*` files.

I think we should continue this, and instead use `libandroid-mono-api.so` or something like that.